spec/lang/step/intrinsics.md

```diff
@@ -117,6 +117,21 @@ impl<M: Memory> Machine<M> {
 }
 ```
 
+`Abort` stopts the machine immediately.
+
+```rust
+impl<M: Memory> Machine<M> {
+    fn eval_intrinsic(
+        &mut self,
+        IntrinsicOp::Abort: IntrinsicOp,
```

Comment:
We should probably remove the `Panic` intrinsic while we are at it. You can just replace all its existing uses with `Abort`.